record[] lines  # lista dei record di correlazione